<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeDialog</class>
 <widget class="QDialog" name="NoticeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="containerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>680</width>
     <height>486</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>14</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>19</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="dialogTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="bgWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
  </widget>
  <zorder>bgWidget</zorder>
  <zorder>containerWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
